remote-backup/run.sh

```diff
@@ -162,9 +164,9 @@ function copy-backup-to-remote {
     fi
 
     bashio::log.info "Copying backup using SFTP/SCP."
-    if ! sshpass -p "${REMOTE_PASSWORD}" scp -s -F "${SSH_HOME}/config" "/backup/${SLUG}.tar" "${REMOTE_HOST}":"${remote_directory}/${remote_name}.tar"; then
+    if ! sshpass -p "${REMOTE_PASSWORD}" scp "${SSH_OPTIONS}" -s -F "${SSH_HOME}/config" "/backup/${SLUG}.tar" "${REMOTE_HOST}":"${remote_directory}/${remote_name}.tar"; then
```

Comment:
This could introduce conflicting options by users which then generate support requests. Overall, I do not really understand how you intend to solve your issue in https://github.com/ikifar2012/remote-backup-addon/issues/76 like this. The keys are loaded from a location specified in the config so overwriting here makes it more complicated. Solution should target the generation of the config, in my opinion.